### PR TITLE
Feature/okn 1791

### PR DESCRIPTION
**Pull request by cristain7**

### CODE REVIEW

File: src/atoms/icons/icon/icon.sass [hunk 1]
```diff
@@ -1,7 +1,3 @@
 .osg-icon
   width: 31px
   height: 31px
-  +osg-mq-tablet-and-up
-    width: 50px
```
Comment (simonerlandsen) [inline src/atoms/icons/icon/icon.sass]: this should probably be added back somewhere to make the front page services list work on bigger screens.

File: src/molecules/navigation/menu_list/menu_list.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    "dataattrs": [
+      { "key": "id", "val": "lorem_ipsum" },
+      { "key": "aria-expanded", "val": "true" }
+    ],
+    "items": [
+      {
+        "icon": {
+          "modifiers": "osg-u-icon-dognapne-vakttjenester"
+        },
+        "text": "Døgnåpne tjenester",
```
Comment (simonerlandsen) [inline src/molecules/navigation/menu_list/menu_list.json]: don't add real data to patterns.

File: src/molecules/navigation/menu_list/menu_list.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    "header": "Tjenester",
+    "modifiers": "osg-menu-list--expandable osg-menu-list--expanded",
+    "dataattrs": [
+      { "key": "id", "val": "lorem_ipsum" },
+      { "key": "aria-expanded", "val": "true" }
+    ],
+    "items": [
+      {
+        "icon": {
+          "modifiers": "osg-u-icon-dognapne-vakttjenester"
```
Comment (simonerlandsen) [inline src/molecules/navigation/menu_list/menu_list.json]: use the new names.

File: src/molecules/navigation/menu_list/menu_list.json (hunk 2 above)
Comment (cristain7) [reply src/molecules/navigation/menu_list/menu_list.json]: Done.

File: src/molecules/navigation/menu_list/menu_list.json (hunk 3 above)
Comment (cristain7) [reply src/molecules/navigation/menu_list/menu_list.json]: I used the default icon chevron-right

File: src/atoms/icons/icon/icon.sass (hunk 1 above)
Comment (cristain7) [reply src/atoms/icons/icon/icon.sass]: maybe add that value  to +osg-mq-fullhd-and-up?

**Comment by simonerlandsen** [comment]
ok

**Pull request closed without merging**